requirements/base.txt

```diff
@@ -2,7 +2,7 @@ chevron~=0.12
 click~=8.0
 Flask<2.3
 #Need to add Schemas latest SDK.
-boto3>=1.19.5,==1.*
+boto3==1.*
```

Comment:
I don't think this is what we want. Don't we need a lower bound due to new APIs we use?

Reply:
I agree. From what I remember the minimum requirement of boto3 was from SAM-T in which `boto3>=1.19.5,==1.*` is already specified. Dependabot keeps updating this. Let me revert this change in this PR and update dependabot config via another PR.

Reply:
I would have figured this comes from SAM CLI than SAM-T. SAM CLI needs things like AWS Signer Apis, AWS Lambda Layers, Schemas, etc, which SAM-T has a couple but nothing recently released (since it translates to CloudFormation). 

Just pointing out, this is like SAM CLI's min requirement not SAM-T. If SAM-T needs a min version, I would expect their requirements to define it.